tabs/onboard_logging.js

```diff
@@ -10,7 +10,14 @@ TABS.onboard_logging.initialize = function (callback) {
     var 
         self = this,
         saveCancelled, eraseCancelled;
+    newFC.purge(); // This should be noved to a general "change-tab" callback
 
+    BLACKBOX.listen(load_html);
+    DATAFLASH.listen(load_html);
+    PID_ADVANCED_CONFIG.listen(load_html);
```

Comment:
Using the listener registration pattern will cause most or all tabs to try and update their display whenever new data is received. This is going to create a lot of side effects (like tabs being rendered). We should really introduce a way to only fire the listener for the currently active tab, not all of them.